pages/common/mysqldump.md

```diff
@@ -11,3 +11,7 @@
 - Backup all databases redirecting the output to a file (user will be prompted for a password):
 
 `mysqldump --user {{user}} --password --all-databases > {{path/to/file.sql}}`
+
+- Backup all databases from a remote host and redirecting the output to a file (user will be prompted for a password):
+
+`mysqldump --host={(ip)} --user {{user}} --password --all-databases > ({path/to/file.sql}}`
```

Comment:
```suggestion
`mysqldump --host={(ip_or_hostname)} --user {{user}} --password --all-databases > ({path/to/file.sql}}`
```

Reply:
Thanks for adding the hostname, I will add the single backup and other things.